openvpn-formula/openvpn-formula.changes

```diff
@@ -0,0 +1,4 @@
+-------------------------------------------------------------------
+Mon Apr 27 14:35:26 UTC 2020 - Alexander Graul <agraul@suse.com>
```

Comment:
Minor thing: should we maybe use are recent date?

Reply:
I can update it to today

Reply:
Updated (ran `osc vc` in another OBS directory and copied the line over)